tests/test-webapps/test-servlet-spec/test-spec-webapp/pom.xml

```diff
@@ -225,5 +225,10 @@
       <!-- deliberately old version to test classloading -->
       <version>9.3.0.RC0</version>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-util</artifactId>
+      <version>${project.version}</version>
```

Comment:
Note the dependency above about deliberately using an old version of jetty-util!

Reply:
Hmmm if sanitize in not in the old version you may have to inline it.

Reply:
Perhaps we need a standalone war / webapp that specifically tests old versions of jetty-util
one that isn't part of demo-base.
one that is only used during integration testing in the build.